src/visual.ts

```diff
@@ -1362,17 +1446,79 @@ module powerbi.extensibility.visual {
             this.root = null;
         }
 
+        private findDatPointCloserToInterval(
+            findMax: boolean,
+            limit: number,
+            settingVal: number,
+            interval: number
+        ): number {
+            while (findMax ? limit > settingVal && settingVal <= limit - interval : limit < settingVal && settingVal >= limit + interval) {
```

Comment:
Please simplify this condition, it's hard to understand how it works.

Reply:
Got it!